js/gallery/src/Gallery.svelte

```diff
@@ -328,6 +327,10 @@
 		border-color: var(--border-color-accent);
 	}
 
+	.thumbnail-item.selected {
+		border: 1px solid var(--color-accent);
```

Comment:
I think we should use border-color-accent here? The hover and border color don't match and I think it would look better if they did

<img width="650" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/f899c2b6-cd65-4ff5-a939-945f047b97d9">